### DISCUSSION TITLE
Wrong Date Display in Stats?

### DISCUSSION THREAD

**Discussion by thehoffm** [YOURLS/YOURLS#3380] [Q&A]
![image](https://user-images.githubusercontent.com/105566590/177826534-5ca8bd94-1e38-4331-95b7-f5881c2a261f.png)

I was looking at the stats from one of my links for the hits in the last 7 days and noticed that it's registering hits from June?

Any ideas?

**Comment by ozh** [comment]
Hello

Cannot reproduce : if I look at a link with no hits in the past week, I see old dates
![image](https://user-images.githubusercontent.com/223647/177997781-e7dc5ef3-ec6c-41b9-aebb-61796254b2ec.png)

You sure your server has the correct date/time ?

**Reply by thehoffm** [reply]
A server issue is what I was thinking initially but new clicks come up as current with the correct timestamp.

But maybe everything is working as intended - you just said if you look at a link with no hits in the past week, you see old dates.
Shouldn't that register as "no hits in the past 7 days" instead of seeing older data?

**Comment by ozh** [comment]
It mostly sort of works this way, see for instance https://yourls.org/d8+ (it says there was 1 hit in the last 7 days in the table next to graphs, I suspect bad code here). If you happen to dig into the code and see something to fix, a pull request is welcome of course :)
